extensions/wikia/PhalanxII/hooks/PhalanxHooks.class.php

```diff
@@ -133,10 +133,6 @@ static public function onEditPhalanxBlock( &$data ) {
 
 		$phalanx['type'] = $typemask;
 
-		if ( $phalanx['lang'] == 'all' ) {
-			$phalanx['lang'] = null;
-		}
-
 		if ( $phalanx['expire'] === '' || is_null( $phalanx['expire'] ) ) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "expire" 7 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)